models/intel/smartlab-action-recognition-0001/smartlab-action-recognition-0001-decoder/README.md

```diff
@@ -0,0 +1,34 @@
+# smartlab-action-recognition-0001 (decoder)
```

Comment:
We usually keep only one readme for composite models. For example https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/intel/text-spotting-0005

Reply:
@Wovchena If I didn't separate readme, I will add .gitkeep file in these empty directories. Is it OK?

Reply:
Can you remove such folders instead? Merging https://github.com/openvinotoolkit/open_model_zoo/pull/3572 will recreate them.

Reply:
Sure, removed.